lua/core/default_config.lua

```diff
@@ -66,8 +65,10 @@ M.options = {
    ruler = false,
    updatetime = 250,
    -- used for updater
-   update_url = "https://github.com/NvChad/NvChad",
```

Comment:
this made more sense imo , nv_chad doesnt :p

Reply:
Fair, happy to change it back,

I do think having a separate section for internal NvChad options could be good, but happy to revert this

Reply:
Changed back, can always revisit later if we need to